src/c2pa/c2pa.py

```diff
@@ -1273,6 +1287,20 @@ def resource_to_stream(self, uri: str, stream: Any) -> int:
 class Signer:
     """High-level wrapper for C2PA Signer operations."""
 
+    # Class-level error messages to avoid multiple creation
+    _ERROR_MESSAGES = {
```

Comment:
Moved all error message strings to class level, so they're not recreated.
Also use them from there with `<class_name>._ERROR_MESSAGES['error_message_key']`.